### PR TITLE
DNM tests - Added test for ceph-deploy + systemd

### PR DESCRIPTION
**Pull request by yuriw**
Upgrade will work luminous => master for now,
and later master will have to be replaced with `mimic`

Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

### CODE REVIEW

File: qa/suites/upgrade/luminous-x/ceph-deploy/luminous-mimic.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+meta:
+- desc: |
+    Setup 4 node ceph cluster using ceph-deploy, use latest
+    stable luminous as initial release, upgrade to mimic (ATM to master) and
+    also setup mgr nodes along after upgrade, check for
+    cluster to reach healthy state, After upgrade run kernel tar/untar
+    task and systemd task. This test will detect any
+    ceph upgrade issue and systemd issues.
```
Comment (amitkumar50) [inline qa/suites/upgrade/luminous-x/ceph-deploy/luminous-mimic.yaml]: should be 'ceph upgrade issues'.

File: qa/suites/upgrade/luminous-x/ceph-deploy/luminous-mimic.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        osd sloppy crc: true
+      client:
+        rbd default features: 5
+openstack:
+- machine:
+    disk: 100
+- volumes:
+    count: 3
+    size: 30
+#  reluctantely :( hard-coded machine type
```
Comment (amitkumar50) [inline qa/suites/upgrade/luminous-x/ceph-deploy/luminous-mimic.yaml]: reluctantely wrong spelling

File: qa/suites/upgrade/luminous-x/ceph-deploy/luminous-mimic.yaml (hunk 1 above)
Comment (yuriw) [reply qa/suites/upgrade/luminous-x/ceph-deploy/luminous-mimic.yaml]: @amitkumar50 fixed on master but will not backport :)

**Comment by yuriw** [comment]
@vasukulkarni I commented out work loads for testing purposes for the time being.  Suspect issues in `ceph-deploy` task
http://pulpito.ceph.com/yuriw-2017-10-16_17:17:18-upgrade:luminous-x:ceph-deploy-master-distro-basic-smithi/

**Comment by yuriw** [comment]
Marked this PR as DNM as unclear if we need to keep testing ceph-deploy on mimic
@vasukulkarni FYI

**Pull request closed without merging**